tests/foreman/api/test_contentaccess.py

```diff
@@ -0,0 +1,75 @@
+# -*- encoding: utf-8 -*-
+"""Test for Content Access (Golden Ticket) API
+
+:Requirement: Content Access
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.decorators import (
+    run_only_on,
+    stubbed,
+    tier2,
+)
+from robottelo.test import APITestCase
+
+
+class ContentAccessTestCase(APITestCase):
+    """Content Access API tests."""
+
+    @classmethod
+    def setUp(self):
+        """Setup must ensure there is an Org with Golden Ticket enabled.
+
+        SQL::
+
+            UPDATE
+                 cp_owner
+            SET
+                 content_access_mode = 'org_environment',
+                 content_access_mode_list='entitlement,org_environment'
+            WHERE account='{org.label}';
```

Comment:
Does it mean that we have in plans to introduce code that will work with SQL database directly?